packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/plugin-ledger-connector-fabric.ts

```diff
@@ -307,15 +307,22 @@ export class PluginLedgerConnectorFabric
     this.sshDebugOn = opts.sshDebugOn === true;
     if (this.opts.sshConfig) {
       this.sshConfig = this.opts.sshConfig;
+
+      if (this.sshDebugOn) {
+        this.sshConfig = this.enableSshDebugLogs(this.sshConfig);
+      }
     } else if (this.opts.sshConfigB64) {
       const sshConfigBuffer = Buffer.from(this.opts.sshConfigB64, "base64");
       const sshConfigString = sshConfigBuffer.toString("utf-8");
       this.sshConfig = JSON.parse(sshConfigString);
+
+      if (this.sshDebugOn) {
+        this.sshConfig = this.enableSshDebugLogs(this.sshConfig);
+      }
     } else {
-      throw new Error("Cannot instantiate Fabric connector without SSH config");
-    }
-    if (this.sshDebugOn) {
-      this.sshConfig = this.enableSshDebugLogs(this.sshConfig);
+      // throw new Error("Cannot instantiate Fabric connector without SSH config");
```

Comment:
Before this comment please add this
```
TODO: Temporarily commenting this code so that we do not have breaking changes, will be fixed by ISSUE_NUM
```

Reply:
I think you straightaway copy pasted my line even though we discussed in the meeting to create a new task to fix this and then replace it with `ISSUE_NUM`.

Reply:
+1 raynatopedrajeta The comment is almost spot on, just please make sure to open an issue and put the number of that issue there in place of `ISSUE_NUM`